config.m4

```diff
@@ -56,6 +56,7 @@ if test "$PHP_DDTRACE" != "no"; then
     src/ext/signals.c \
     src/ext/span.c \
     src/ext/third-party/mt19937-64.c \
+    src/ext/integrations/integrations.c \
```

Comment:
For a pecl build I think you also need to add the directory; something like `PHP_ADD_BUILD_DIR([$ext_builddir/src/ext/integrations])`. You might also need to add it as an include dir too `PHP_ADD_INCLUDE([$ext_srcdir/src/ext/mpack])`.